<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>475</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Configurar Simulación</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="botellas">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>481</width>
      <height>221</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>155</number>
    </attribute>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Cantidad de Refresco</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Probabilidad de Colocar</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Probabilidad de Desecho</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>25</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>25</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>25</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>30</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>25</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>30</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>40</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>25</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>20</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="cantidadRefresco">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="nuevaBotella">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Insertar Botella</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="probabilidadDesecho">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cantidad Refresco</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Probabilidad de Desecho</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Probabilidad de Colocar</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="probabilidadColocar">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tiempo de Colocación</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tiempoColocacion">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>70</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>110</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tiempo de Limpieza</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="tiempoLimpieza">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>130</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tiempoLlenado1">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>170</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tiempo de Llenado 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>290</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tiempo de Entapado</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="tiempoEntapado">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>310</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>350</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tiempo de Inspección</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="tiempoInspeccion">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>370</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>50</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Capacidad Banda Colocadas</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="capacidadColocadas">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>70</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>110</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Capacidad Banda Limpias 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>230</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Capacidad Banda Llenas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>290</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Capacidad Banda Entapadas</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="capacidadLimpias1">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>130</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="capacidadLlenas">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>250</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="capacidadEntapadas">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>310</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Inicio">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>340</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Iniciar Simulación</string>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/botones/botones/Flecha Derecha.jpg</normaloff>:/botones/botones/Flecha Derecha.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>541</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ingrese los parámetros que utilizará la simulación</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinEliminar">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>380</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="eliminarBotella">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>380</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar Botella</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>170</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Capacidad Banda Limpias 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="capacidadLimpias2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>190</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>230</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tiempo de Llenado 2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="tiempoLlenado2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>250</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>922</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>cantidadRefresco</tabstop>
  <tabstop>probabilidadColocar</tabstop>
  <tabstop>probabilidadDesecho</tabstop>
  <tabstop>nuevaBotella</tabstop>
  <tabstop>botellas</tabstop>
 </tabstops>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>nuevaBotella</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>agregarBotella()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eliminarBotella</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>eliminarBotella()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Inicio</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>iniciarSimulacion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>831</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>887</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>agregarBotella()</slot>
  <slot>eliminarBotella()</slot>
  <slot>iniciarSimulacion()</slot>
 </slots>
</ui>
